pkg/arvo/app/soto.hoon

```diff
@@ -0,0 +1,159 @@
+::  Soto: A Dojo relay for Urbit's Landscape interface
+::  Relays sole-effects to subscribers and forwards sole-action pokes
+/-  sole
+/+  *server, *sole
+/=  index
+  /^  octs
+  /;  as-octs:mimes:html
+  /:  /===/app/soto/index
+  /|  /html/
+      /~  ~
+  ==
+/=  tile-js
+  /^  octs
+  /;  as-octs:mimes:html
+  /:  /===/app/soto/js/tile
+  /|  /js/
+      /~  ~
+  ==
+/=  script
+  /^  octs
+  /;  as-octs:mimes:html
+  /:  /===/app/soto/js/index
+  /|  /js/
+      /~  ~
+  ==
+/=  style
+  /^  octs
+  /;  as-octs:mimes:html
+  /:  /===/app/soto/css/index
+  /|  /css/
+      /~  ~
+  ==
+/=  soto-png
+  /^  (map knot @)
+  /:  /===/app/soto/img  /_  /png/
+::
+=,  format
+|%
++$  state
+  $%  [%0 bon=bone]                                     :: store bone from Dojo peer
```

Comment:
```suggestion
  ::  bon: bone from dojo peer
  ::
  $%  [%0 bon=bone]
```

Note also the double-space after `::`.

There's margin comments in other places too, let's full-line those as well.